grpc/rust/BUILD

```diff
@@ -0,0 +1,39 @@
+load("@rules_rust//proto:proto.bzl", "rust_grpc_library")
+load("@rules_rust//proto:toolchain.bzl", "rust_proto_toolchain")
+
+rust_proto_toolchain(
```

Comment:
Source: https://bazelbuild.github.io/rules_rust/rust_proto.html#customizing-dependencies
> These rules depends on the [protobuf](https://crates.io/crates/protobuf) and the [grpc](https://crates.io/crates/grpc) crates in addition to the [protobuf compiler](https://github.com/google/protobuf). To obtain these crates, rust_proto_repositories imports the given crates using BUILD files generated with [cargo raze](https://github.com/google/cargo-raze).
>
> If you want to either change the protobuf and gRPC rust compilers, or to simply use [cargo raze](https://github.com/google/cargo-raze) in a more complex scenario (with more dependencies), you must redefine those dependencies.
>
> To do this, once you’ve imported the needed dependencies (see our [Cargo.toml](https://bazelbuild.github.io/rules_rust/raze/Cargo.toml) file to see the default dependencies), you need to create your own toolchain. To do so you can create a BUILD file with your toolchain definition

Reply:
TO REVISIT: Where should this toolchain be kept? Architecturally, it should probably live in `@vaticle_dependencies`.